hail_search/test_search.py

```diff
@@ -219,6 +219,12 @@ async def test_status(self):
             resp_json = await resp.json()
         self.assertDictEqual(resp_json, {'success': True})
 
+    async def test_reload_globals(self):
```

Comment:
Do we want to test that the globals were actually reloaded? It seems like that would be hard to test but I feel like we do want to be sure that something happened. Maybe we mock something?

Reply:
Updated to test with a set of mock globals, and also updated the route to return the globals themselves.